docs/documentation/server_admin/topics.adoc

```diff
@@ -62,6 +62,7 @@ include::topics/admin-console-permissions/fine-grain.adoc[]
 endif::[]
 include::topics/assembly-managing-organizations.adoc[]
 include::topics/assembly-managing-clients.adoc[]
+include::topics/oid4vci/vc-issuer-configuration.adoc[]
```

Comment:
Is it possible to wrap this line within `ifeval::[{project_community}==true]` and `endif::[]`, so this is only in the community documentation? See some other links in the same document for the "inspiration", like for example `fine-grain.adoc` .